packages/at_onboarding_flutter/lib/l10n/intl_zh_Hans_CH.arb

```diff
@@ -0,0 +1,114 @@
+{
+    "activate_an_atSign": "激活一个@符号",
```

Comment:
This doesn't look quite right, `atSign` got translated, make sure it still means "atSign" and not "at sign"

Reply:
Same for other localizations

Reply:
Done, addressed it for all the languages to keep atSign as a proper noun.